support/package_nginx

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env bash
+#
+# This script is used for the [Nginx
+# buildpack](github.com/Scalingo/nginx-buildpack/). Please refer to this page
+# (https://www.notion.so/scalingooriginal/New-Nginx-Version-936b92e173a74f5ab0cd59e5155a4880)
```

Comment:
Can you update this Notion page? Especially the part which talks about how to upload on OVH. Should you create a user for this and add the access and secret key to LastPass?

Reply:
Yes it will be done!